Source/items.cpp

```diff
@@ -3972,89 +3964,89 @@ void UseItem(int p, item_misc_id Mid, spell_id spl)
 	case IMISC_ELIXVIT:
 		ModifyPlrVit(p, 1);
 		if (gbIsHellfire) {
-			plr[p]._pHitPoints = plr[p]._pMaxHP;
-			plr[p]._pHPBase = plr[p]._pMaxHPBase;
+			player._pHitPoints = player._pMaxHP;
+			player._pHPBase = player._pMaxHPBase;
 			drawhpflag = true;
 		}
 		break;
 	case IMISC_REJUV:
-		j = plr[p]._pMaxHP >> 8;
+		j = player._pMaxHP >> 8;
 		l = ((j / 2) + GenerateRnd(j)) << 6;

```

Comment:
Ah someone changed the >> 1 to a / 2 here.
Still the same as above:
```suggestion
		l = GenerateRnd(player._pMaxHP / 4) + player._pMaxHP / 8;
```